lib/mongoid/association/embedded/embeds_many/proxy.rb

```diff
@@ -217,10 +217,11 @@ def exists?
           #   person.addresses.find([ BSON::ObjectId.new, BSON::ObjectId.new ])
           #
           # @param [ Array<Object> ] args Various arguments.
+          # @param [ Proc ] block Optional block to pass.
```

Comment:
Can these docstrings be augmented to 1) either include a copy of  the text added in https://github.com/mongodb/mongoid/pull/4817/files that explains the block usage, or link to the documentation of the methods documented in https://github.com/mongodb/mongoid/pull/4817/files, so that in either case the block usage is documented, and 2) provide an example with the block form since currently the other two invocations are documented with examples (could be only the single line)?

Reply:
Added